lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/aplinear/APLMeanSummarizer.java

```diff
@@ -56,10 +61,10 @@
     public List<QualityMetric> getQualityMetricList() {
         List<QualityMetric> qualityMetricList = new ArrayList<>();
         qualityMetricList.add(
-                new SupportMetric(0)
+                new SupportQualityMetric(0)
```

Comment:
This is more for @edgan8, but the way these QualityMetrics are instantiated seems confusing—what does the "0" mean here? (I know it's actually the index into aggregates, but I don't know why it has to be 0. The constructor isn't actually helpful here.) The QualityMetrics have a lot of implicit conventions that are just assumed to be followed, and there isn't much documentation for these conventions anywhere.